scenarios/change-data-capture/README.md

```diff
@@ -0,0 +1,176 @@
+# Pravega CDC Demo
+
+This page demonstrates how to preserve the CDC (change data capture) from MySQL into Pravega and consume them in Flink.
+
+> Change data capture (CDC) is used to determine and track the data that has changed so that actions can be taken using the changed data. It is often used in data-warehouse environments where we want have a copy or log of the existing data.
+
+We use Pravega as the streaming storage system which can preserve the continuous data stream, so the change data can be both written to and read from a durable, elastic and consistent place. Also, the debezium json format is used as the intermediate format when we transfer the data. You may have a look at [Pravega](https://cncf.pravega.io/) and [Debezium](https://debezium.io/) respectively at their homepage.
+
+## Preparation
+
+The demo runs under `docker-compose`, so a Linux or Windows (WSL2) environment is required.
+
+With `docker` enabled, download the [`docker-compose.yml`](./docker-compose.yml) and put it anywhere you like.
+
+After having the `docker-compose.yml`, start it like this:
+
+```bash
+docker-compose up -d
+```
+
+This command automatically starts all the containers defined in the docker compose configuration in the detached mode. Run `docker ps` if you wish to check whether the 7 containers are running properly. `http://localhost:12081/` is also available for Flink UI.
```

Comment:
"This command automatically starts *in detached mode* all the containers defined in the docker compose configuration."